test/e2e/util/hypershift_framework.go

```diff
@@ -128,7 +159,8 @@ func (h *hypershiftTest) before(hostedCluster *hyperv1.HostedCluster, opts *Plat
 }
 
 // runs after each test.
-func (h *hypershiftTest) after(hostedCluster *hyperv1.HostedCluster, platform hyperv1.PlatformType) {
+func (h *hypershiftTest) after(platform hyperv1.PlatformType) {
```

Comment:
unrelated refactor (along with changes to `teardown` and `postTeardown` signatures)